backend/services/deployments/app/app.go

```diff
@@ -1852,6 +1854,21 @@ func (d *Deployments) LookupDeployment(ctx context.Context,
 	return list, totalCount, nil
 }
 
+func (d *Deployments) LookupDeploymentV2(ctx context.Context,
+	query model.Query) ([]*model.Deployment, error) {
+	list, err := d.db.FindDeployments(ctx, query)
+
+	if err != nil {
+		return nil, errors.Wrap(err, "searching for deployments")
+	}
+
+	if list == nil {
+		return make([]*model.Deployment, 0), nil
```

Comment:
I would write it this way:

```suggestion
		list = make([]*model.Deployment, 0)
```